xDiffCollection/Classes/CollectionBin.swift

```diff
@@ -81,6 +90,29 @@ public struct CollectionBin<T, Backstorage: RangeReplaceableCollection>: Collect
         
         return updating(with: _backstorage.replacing(with: element, at: index ))
     }
+    
+    //Linear insert function assumes that optional _sort is NOT nil
+    private func linearInsert(_ element:T) -> Backstorage {
```

Comment:
you could have passed the sort function here.. would help avoid forced unwrapping